<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Register</class>
 <widget class="QWidget" name="Register">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>485</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="MyWidgetRegister" name="widgetInner" native="true">
     <layout class="QGridLayout" name="gridLayout_4">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QWidget" name="widget_register_nav" native="true">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>403</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_3">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>--</string>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButtonRegisterClose">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="imagelogin.qrc">
             <normaloff>:/Login/C:/Users/ASUS/Pictures/Login/wkf.png</normaloff>:/Login/C:/Users/ASUS/Pictures/Login/wkf.png</iconset>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="page">
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QWidget" name="widget_2" native="true">
            <layout class="QGridLayout" name="gridLayout_2">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <property name="spacing">
              <number>0</number>
             </property>
             <item row="0" column="0">
              <widget class="QLabel" name="label_5">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>40</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>楷体</family>
                 <pointsize>20</pointsize>
                </font>
               </property>
               <property name="text">
                <string>注册账号</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QWidget" name="widget" native="true">
               <widget class="QLabel" name="label">
                <property name="geometry">
                 <rect>
                  <x>60</x>
                  <y>65</y>
                  <width>54</width>
                  <height>12</height>
                 </rect>
                </property>
                <property name="text">
                 <string>用户名</string>
                </property>
               </widget>
               <widget class="QLabel" name="label_2">
                <property name="geometry">
                 <rect>
                  <x>60</x>
                  <y>115</y>
                  <width>54</width>
                  <height>12</height>
                 </rect>
                </property>
                <property name="text">
                 <string>密码</string>
                </property>
               </widget>
               <widget class="QLabel" name="label_3">
                <property name="geometry">
                 <rect>
                  <x>60</x>
                  <y>165</y>
                  <width>54</width>
                  <height>12</height>
                 </rect>
                </property>
                <property name="text">
                 <string>确认密码</string>
                </property>
               </widget>
               <widget class="QLineEdit" name="lineEdit_user_name">
                <property name="geometry">
                 <rect>
                  <x>140</x>
                  <y>60</y>
                  <width>211</width>
                  <height>25</height>
                 </rect>
                </property>
                <property name="placeholderText">
                 <string>请输入用户名</string>
                </property>
               </widget>
               <widget class="QLineEdit" name="lineEdit_pwd">
                <property name="geometry">
                 <rect>
                  <x>140</x>
                  <y>110</y>
                  <width>211</width>
                  <height>25</height>
                 </rect>
                </property>
                <property name="echoMode">
                 <enum>QLineEdit::Password</enum>
                </property>
                <property name="cursorPosition">
                 <number>0</number>
                </property>
                <property name="placeholderText">
                 <string>请输入密码</string>
                </property>
               </widget>
               <widget class="QLineEdit" name="lineEdit_sure_pwd">
                <property name="geometry">
                 <rect>
                  <x>140</x>
                  <y>160</y>
                  <width>211</width>
                  <height>25</height>
                 </rect>
                </property>
                <property name="echoMode">
                 <enum>QLineEdit::Password</enum>
                </property>
                <property name="placeholderText">
                 <string>请确认密码</string>
                </property>
               </widget>
               <widget class="QPushButton" name="pushButtonRegister">
                <property name="geometry">
                 <rect>
                  <x>70</x>
                  <y>350</y>
                  <width>280</width>
                  <height>30</height>
                 </rect>
                </property>
                <property name="cursor">
                 <cursorShape>PointingHandCursor</cursorShape>
                </property>
                <property name="text">
                 <string>注册</string>
                </property>
               </widget>
               <widget class="QLabel" name="label_4">
                <property name="geometry">
                 <rect>
                  <x>60</x>
                  <y>210</y>
                  <width>54</width>
                  <height>12</height>
                 </rect>
                </property>
                <property name="text">
                 <string>性别</string>
                </property>
               </widget>
               <widget class="QRadioButton" name="radioButton_man">
                <property name="geometry">
                 <rect>
                  <x>140</x>
                  <y>210</y>
                  <width>89</width>
                  <height>16</height>
                 </rect>
                </property>
                <property name="text">
                 <string>男</string>
                </property>
               </widget>
               <widget class="QRadioButton" name="radioButton_woman">
                <property name="geometry">
                 <rect>
                  <x>240</x>
                  <y>210</y>
                  <width>89</width>
                  <height>16</height>
                 </rect>
                </property>
                <property name="text">
                 <string>女</string>
                </property>
               </widget>
               <widget class="QLabel" name="label_6">
                <property name="geometry">
                 <rect>
                  <x>60</x>
                  <y>255</y>
                  <width>54</width>
                  <height>12</height>
                 </rect>
                </property>
                <property name="text">
                 <string>电话</string>
                </property>
               </widget>
               <widget class="QLineEdit" name="lineEdit_phone">
                <property name="geometry">
                 <rect>
                  <x>140</x>
                  <y>250</y>
                  <width>211</width>
                  <height>25</height>
                 </rect>
                </property>
                <property name="placeholderText">
                 <string>请输入电话</string>
                </property>
               </widget>
               <widget class="QLabel" name="label_9">
                <property name="geometry">
                 <rect>
                  <x>60</x>
                  <y>300</y>
                  <width>54</width>
                  <height>12</height>
                 </rect>
                </property>
                <property name="text">
                 <string>邮箱</string>
                </property>
               </widget>
               <widget class="QLineEdit" name="lineEdit_email">
                <property name="geometry">
                 <rect>
                  <x>140</x>
                  <y>299</y>
                  <width>211</width>
                  <height>25</height>
                 </rect>
                </property>
                <property name="placeholderText">
                 <string>请输入邮箱</string>
                </property>
               </widget>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_2">
         <widget class="QLabel" name="label_7">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>20</y>
            <width>440</width>
            <height>20</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>楷体</family>
            <pointsize>16</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>注册成功</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
         <widget class="QLabel" name="label_8">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>100</y>
            <width>54</width>
            <height>12</height>
           </rect>
          </property>
          <property name="text">
           <string>用户名:</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_user_name">
          <property name="geometry">
           <rect>
            <x>170</x>
            <y>100</y>
            <width>221</width>
            <height>16</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">color:red;</string>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_10">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>170</y>
            <width>54</width>
            <height>12</height>
           </rect>
          </property>
          <property name="text">
           <string>密码:</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_pwd">
          <property name="geometry">
           <rect>
            <x>170</x>
            <y>170</y>
            <width>221</width>
            <height>16</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">color:red;</string>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_close">
          <property name="geometry">
           <rect>
            <x>200</x>
            <y>310</y>
            <width>75</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>关闭</string>
          </property>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyWidgetRegister</class>
   <extends>QWidget</extends>
   <header location="global">mywidgetregister.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="imagelogin.qrc"/>
 </resources>
 <connections/>
</ui>
